logstash-core/src/test/java/org/logstash/secret/store/backend/JavaKeyStoreTest.java

```diff
@@ -473,6 +473,7 @@ public void testExternalUpdateRead() throws IOException {
      */
     @Test
     public void testFileLock() throws Exception {
+        boolean isWindows = System.getProperty("os.name").startsWith("Windows");
```

Comment:
@jakelandis This is a really confusing test now, since assertions are wrapped in conditionals.
Maybe just split it into two tests, `testFileLockUnix` and `testFileLockWindows` and skip the one that doesn't apply for the test running system via `Assume`?

Reply:
I removed an uneeded isWindows? assertion which only leaves 1 isWindows? assertion.    

There is now only 2 lines (out of 40ish) that are windows only, with a comment as to why. I will leave the single test, and think it is a bit less confusing now.

Reply:
@jakelandis sure makes sense :)